src/components/MapView.js

```diff
@@ -279,18 +279,13 @@ class MapView extends Component {
                         </div>
                     </div>
                     {this.showListViewButton(isMobile, classes, history)}
-                    <div >
-                        <Dialog
-                            open={legend}
-                            onClose={this.handleClose}
-                            className="two-col-special"
-                        >
-                            <DialogContent  >
-                                <div >
-                                    {this.showLegend()}
-                                </div>
-                            </DialogContent>
-                        </Dialog>
+                    <div>
+                        <div className={legend ? "legend-open" : "legend-closed"}>
```

Comment:
This is a super-small detail, but our mocks have the legend open by default on desktop, but closed by default on mobile. Would you mind updating the initial state to reflect this?